crates/bevy_reflect/src/type_uuid.rs

```diff
@@ -7,6 +7,8 @@ pub trait TypeUuid {
 
 pub trait TypeUuidDynamic {
     fn type_uuid(&self) -> Uuid;
+    /// Helper to display the type in a readable manner to the user
+    fn display_type(&self) -> String;
 }
 
 impl<T> TypeUuidDynamic for T
```

Comment:
Is there a specific reason you didn't use the `std::fmt::Display` trait instead? That would allow you to, among other things, use the `to_string()` method up above.

Reply:
Actually, I don't think you'll even need to use `to_string()` in that case.

Reply:
I didn't check, but I think I can't implement `Display` for this trait, as it may conflict with existing implementation for `TypeUuidDynamic` implementors